core/src/main/java/io/questdb/std/Unsafe.java

```diff
@@ -193,6 +194,17 @@ public static long getMemUsedByTag(int memoryTag) {
         return COUNTERS[memoryTag].sum();
     }
 
+    @TestOnly
+    public static String getMemUsedHumanReadable() {
+        StringBuilder sb = new StringBuilder();
+
+        for (int i = 0, n = COUNTERS.length; i < n; i++) {
+            sb.append(MemoryTag.nameOf(i)).append(": ").append(COUNTERS[i]).append("\n");
```

Comment:
Call `getMemUsedByTag` instead of accessing `COUNTERS` directly: This will prevent merge conflict issues with https://github.com/questdb/questdb/pull/4989

Reply:
For context, the new impl will read:
```java
    public static long getMemUsedByTag(int memoryTag) {
        assert memoryTag >= 0 && memoryTag < MemoryTag.SIZE;
        return COUNTERS[memoryTag].sum() + UNSAFE.getLongVolatile(null, NATIVE_MEM_COUNTER_ADDRS[memoryTag]);
    }
```